Source/References/PixelRendr-0.2.0.ts

```diff
@@ -758,6 +765,24 @@ module PixelRendr {
         */
 
         /**
+         * Replaces the current palette with a new one.
+         * 
+         * @param palette    The new palette to replace the current one.
+         */
+        changePalette(palette: IPalette): void {
+            this.paletteDefault = palette;
+            this.digitsizeDefault = this.getDigitSizeFromArray(this.paletteDefault);
+            this.digitsplit = new RegExp(`.{1,${this.digitsizeDefault}}`, "g");
```

Comment:
This is copy + pasted from the constructor. In the interest of not duplicating code let's move it to a private method.